ckanext/chevrontheme/templates/snippets/package_item.html

```diff
@@ -56,7 +60,7 @@
       {% endif %}
     </td>
     <td>
-      {{ package.quarantine_is_species_native_to_area }}
```

Comment:
this one should be conditional to, you need to fix all the field changes if they're specific to Padil so you don't break the table view for the other data types.

Reply:
Fixed.